spec/lib/redfin_spec.rb

```diff
@@ -1,15 +1,72 @@
 require 'spec_helper'
 
-RSpec.describe SherlockHomes::Redfin, sauce: true do
+RSpec.describe SherlockHomes::Redfin, skip_ci: true do
 
-  Given(:scraper) do
-    subject.base_url 'https://www.redfin.com/PA/Allentown/2354-S-Cedar-Crest-Blvd-18103/home/59034427'
-    subject
+  When do
+    subject.class.set_url 'https://www.redfin.com/PA/Allentown/2354-S-Cedar-Crest-Blvd-18103/home/59034427'
+    subject.load
+    subject.wait_for_contact_box
```

Comment:
What is: wait_for_contact_box?  I'm assuming it's an ajax thing that we need to wait for before finishing pulling all of the data. I'd prefer to bake this into the scraper though.